kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/models/ItemViewBindingEpoxyHolder.kt

```diff
@@ -0,0 +1,20 @@
+package com.airbnb.epoxy.kotlinsample.models
+
+import com.airbnb.epoxy.EpoxyAttribute
+import com.airbnb.epoxy.EpoxyModelClass
+import com.airbnb.epoxy.kotlinsample.R
+import com.airbnb.epoxy.kotlinsample.databinding.ViewBindingHolderItemBinding
+import com.airbnb.epoxy.kotlinsample.helpers.ViewBindingEpoxyModel
+
+@EpoxyModelClass(layout = R.layout.view_binding_holder_item)
+abstract class ItemViewBindingEpoxyHolder :
```

Comment:
I'm a bit rusty on the kotlin epoxy model approach, but I believe it would be possible to combine it with the viewholder approach, so we would get this improved ease of binding along with being able to use kotlin data classes instead of generated models off of annotations

Reply:
nevermind, I don't know that that made sense